### ISSUE TITLE
Having Trouble Scraping the Timeline

### ISSUE THREAD

**Issue by deadbeatdandylyon** [prof79/fansly-downloader-ng#96]
Hello all, Was wondering if someone could kindly assist. 

I am new to this scraper and yes I am aware of the warning signs of using this. 

I have everything configured in the scraper however after about 2 videos and one image that got scraped I received the following message

[ [43]ERROR ] [2024-10-16 | 01:12]: Unexpected error during Timeline download: 
Traceback (most recent call last):
  File "download\common.py", line 151, in process_download_accessible_media
  File "download\media.py", line 202, in download_media
  File "fileio\dedupe.py", line 74, in dedupe_media_file
  File "fileio\fnmanip.py", line 71, in get_hash_for_image
  File "imagehash\__init__.py", line 272, in phash
  File "PyInstaller\loader\pyimod02_importers.py", line 419, in exec_module
  File "scipy\fftpack\__init__.py", line 93, in <module>
  File "PyInstaller\loader\pyimod02_importers.py", line 419, in exec_module
  File "scipy\fftpack\_basic.py", line 8, in <module>
  File "PyInstaller\loader\pyimod02_importers.py", line 419, in exec_module
  File "scipy\fft\__init__.py", line 91, in <module>
  File "PyInstaller\loader\pyimod02_importers.py", line 419, in exec_module
  File "scipy\fft\_fftlog.py", line 10, in <module>
  File "PyInstaller\loader\pyimod02_importers.py", line 419, in exec_module
  File "scipy\fft\_fftlog_backend.py", line 4, in <module>
  File "PyInstaller\loader\pyimod02_importers.py", line 419, in exec_module
  File "scipy\special\__init__.py", line 807, in <module>
  File "scipy\\special\\_ufuncs.pyx", line 17294, in init scipy.special._ufuncs
ModuleNotFoundError: No module named 'scipy.special._special_ufuncs'

I disabled by NordVPN thinking that might be the problem but the issue still happened, not sure if I'm missing something else?

**Comment by palmerbomber1** [comment]
Im guessing you are using the windows exe to run this? From my understanding having looked through the other issues, a package was not packed into the exe, and I havent found a solution to it yet. Supposedly running it in a python environment makes it so this isnt an issue, but personally I have not been able to get it installed on my pc yet

**Comment by deadbeatdandylyon** [comment]
First of all thank you very much for your reply, Yes this was an attempt at running the application through the .EXE application. 

I did also attempt to run through python method however I have ran into nothing but issues with the python method when it comes to installing the requirements.txt and a few others, i've tried looking through but there really isn't any clear walk through guide on how to set it up properly

**Comment by palmerbomber1** [comment]
I buckled down and tried getting it running again. I also cant seem to get the command to install by requirements.txt to work, but manually going through and installing each element manually with `pip install [requirement]` worked for me, and its no longer pausing on each image like the exe does

**Comment by deadbeatdandylyon** [comment]
I just now attempted to run the python route, 

When I tried to run the `pip install -r requirements.txt` I got the following return error

![image](https://github.com/user-attachments/assets/c90d37d2-121b-435d-80eb-d4243ebc8a24)

I am running version 3.13.0 for python

**Comment by deadbeatdandylyon** [comment]
Well I got the above issue resolved, the problem was is that I had to install python version 3.11 so once that installed I was able to run the requirements 

Now my issue is when I run `.\fansly_downloader_ng.py` the window just opens and closes

**Comment by deadbeatdandylyon** [comment]
Alright everything is resolved! 

A few key points if anyone else has the same problem, I did have to run this through as a python script. 

Even if you have a later version of python installed such as 3.12 or later you still need to have version 3.11 to successfully run the requirements.txt script 

Then I had to manually enter the authorization and user agent and double check the checkkey and once was all verified manually I was successfully able to run as scrape using the "timeline" function  - no errors for images or videos. 

A thank you to @palmerbomber1 for replying and attempting to assist!
